src/emucore/OSystem.cxx

```diff
@@ -183,7 +183,7 @@ void OSystem::resetRNGSeed() {
 
   // We seed the random number generator. The 'time' seed is somewhat redundant, since the
   // rng defaults to time. But we'll do it anyway.
-  if (mySettings->getString("random_seed") == "time") {
+  if (mySettings->getInt("random_seed") == -1) {
```

Comment:
Sure. I'll change it now. Since it is not an unsigned int, I don't see a difference, but it is fine by me if you want it to be 0.